Doc/library/asyncio-task.rst

```diff
@@ -504,6 +516,12 @@ Task
       get_stack().  The file argument is an I/O stream to which the output
       is written; by default output is written to sys.stderr.
 
+   .. attribute:: name
```

Comment:
I care way more about consistency of asyncio APIs than about maintaining any kind of artificial similarity between Task and Thread.  And in asyncio we don't use properties that much. Please replace the `name` property with a `Task.get_name()` method.  There's no use case for adding `set_name()` though, so let's not add it at least in 3.8.

Reply:
Got it.